sql.md

```diff
@@ -0,0 +1,100 @@
+#ORDER:
+
+#Find all subjects sorted by subject
+*SELECT subject FROM subjects
+*ORDER BY subjects ASC;
+
+#Find all subjects sorted by location
+*SELECT subject FROM subjects
+*ORDER BY location ASC;
+
+#WHERE
+
+#Find the book Little Women
+*SELECT * FROM books
+*WHERE title = 'Little Women';
+
+#Find all books containing the word Python
+*SELECT * FROM books
+*WHERE title ILIKE '%python%';
+
+#Find all subjects with the location "Main St" sort them by subject
+*SELECT * FROM subjects
+*WHERE location = 'Main St'
+*ORDER BY subject ASC;
+
+#JOINS
+
+#Find all books about Computers list ONLY book title
+*SELECT title FROM books
+*INNER JOIN subjects
+*ON books.subject_id = subjects.id
+*WHERE subject = 'Computers';
+
+#Find all books and display ONLY
+##Book title
+##Author's first name
+##Author's last name
+##Book subject
+*SELECT books.title, authors.first_name, authors.last_name, subjects.subject  
+*FROM books
+*LEFT JOIN authors
+*ON books.author_id = authors.id
+*INNER JOIN subjects
+*ON books.subject_id = subjects.id;
```

Comment:
alternatively you don't need to specify left joins - select books.title, authors.first_name, authors.last_name, subjects.subject from books join authors on books.author_id = authors.id join subjects on books.subject_id = subjects.id;